shared/js/background/chrome-events.es6.js

```diff
@@ -166,6 +166,17 @@ function blockTrackingCookies () {
     return true
 }
 
+/**
+ * Checks if a tracker is a first party by checking entity data
+ * @param {string} trackerUrl
+ * @param {string} siteUrl
+ * @returns {boolean}
+ */
+function isFirstPartyByEntity (trackerUrl, siteUrl, request) {
+    const trackerData = trackers.getTrackerData(trackerUrl, siteUrl, request)
+    return trackerData ? trackerData.firstParty : utils.isFirstParty(trackerUrl, siteUrl) // fall back on hostname check if trackers is null
```

Comment:
You shouldn't need to fall back here. `trackerData.firstParty` will be true for when eTLDs match because owner will be always the same. Let's drop this function and use `trackers.getTrackerData(trackerUrl, siteUrl, request).firstParty`.

Reply:
as discussed - I read it wrong, the fallback is needed for when something is not a tracker